### ISSUE TITLE
[UBS-User.Edit Profile] The keyboard navigation in the drop-down list does not work.

### ISSUE THREAD

**Issue by oksanakushnir** [ita-social-projects/GreenCity#6097]
**Environment:** all
**Reproducible:** always
**Build found:** 24/07/2023

**Preconditions**
1. The registered user is logged in https://www.greencity.social/#/ubs
2. User data tab is opened

**Steps to reproduce**
1. Click on “Edit data” button
2. Click on "Add address" button
3. Input first letter "K" in "Region" field
4. Choose the keyboard navigation in the drop-down list second region
5. Input first letter "K" in "City" field
6. Choose the keyboard navigation in the drop-down list second city
7. Input first letter "K" in "Street" field
8. Choose the keyboard navigation in the drop-down list second street

**Actual result**
The user can not navigate through the item  drop-down list using keyboard.

![New address](https://github.com/ita-social-projects/GreenCity/assets/130609090/93d247ed-74db-4bfe-a21b-4d7a9fd04f8a)

**Expected result**
The user should be able navigate through the item  drop-down list using keyboard.

**User story and test case links**
E.g.: "User story #3811 

**Labels to be added**
"Bug", Priority ("Medium"), Severity ("Minor"), Type ("Functional")

**Comment by Maksym8Kravchenko** [comment]
This bug is fixed
